### DISCUSSION TITLE
Transclude block links at publishing to be equivalent to Obsidian client rendering of those links

### DISCUSSION THREAD

**Discussion by SemanticBeeng** [GooRoo/mkdocs-obsidian-bridge#8] [Ideas]
Essentially handle

 * `[[embed link#heading]]`
 * `[[embed link#^block]]
 
 See
 * https://help.obsidian.md/Linking+notes+and+files/Internal+links#Link+to+a+block+in+a+note
 * https://github.com/orgs/ObsidianPublisher/discussions/190#discussioncomment-6679194
 * https://obsidian-publisher.netlify.app/template/example/

**Comment by GooRoo** [comment]
I'm sorry for a delayed reply. I had actually read your idea once you wrote it, but I forgot to comment 🙂 
I was not aware of such a syntax at all. Will see what I can do.
